components/automate-cli/pkg/verifyserver/constants/checkconstant.go

```diff
@@ -0,0 +1,59 @@
+package constants
+
+const HARDWARE_RESOURCE_COUNT = "hardware-resource-count"
+const CERTIFICATE = "certificate"
+const SSH_USER = "ssh-user"
+const SYSTEM_RESOURCES = "system-resources"
+const SOFTWARE_VERSIONS = "software-versions"
+const SYSTEM_USER = "system-user"
+const S3_BACKUP_CONFIG = "s3-backup-config"
+const FQDN = "fqdn"
+const FIREWALL = "firewall"
+const EXTERNAL_OPENSEARCH = "external-opensearch"
+const AWS_OPENSEARCH_S3_BUCKET_ACCESS = "aws-opensearch-s3-bucket-access"
+const EXTERNAL_POSTGRESQL = "external-postgresql"
+const NFS_BACKUP_CONFIG = "nfs-backup-config"
+
+func GetAllChecks() []string {
+	var allChecks = []string{
+		HARDWARE_RESOURCE_COUNT,
+		CERTIFICATE,
+		SSH_USER,
+		SYSTEM_RESOURCES,
+		SOFTWARE_VERSIONS,
+		SYSTEM_USER,
+		S3_BACKUP_CONFIG,
+		FQDN,
+		FIREWALL,
+		EXTERNAL_OPENSEARCH,
+		AWS_OPENSEARCH_S3_BUCKET_ACCESS,
+		EXTERNAL_POSTGRESQL,
+		NFS_BACKUP_CONFIG,
+	}
+	return allChecks
+}
+
+func GetBastionChecks() []string {
+	var checks = []string{
+		HARDWARE_RESOURCE_COUNT,
```

Comment:
Instead of this, declare two global slice variables, one holds the bastionChecks and another holds the remoteChecks. And use them to send the response of GetAllChecks, GetBastionChecks and GetRemoteChecks methods.